src/sync/installation.ts

```diff
@@ -327,24 +337,22 @@ const doProcessInstallation = async (data: BackfillMessagePayload, sentry: Hub,
  * Handles an error and takes action based on the error type and parameters
  */
 export const handleBackfillError = async (
-	err,
+	err: Error,
 	data: BackfillMessagePayload,
 	nextTask: Task,
 	rootLogger: Logger,
 	scheduleNextTask: (delayMs: number) => void): Promise<void> => {
 
 	const logger = rootLogger.child({ err });
 
-	const isRateLimitError = err instanceof RateLimitingError || Number(err?.headers?.["x-ratelimit-remaining"]) == 0;
-
-	if (isRateLimitError) {
-		const rateLimit = err instanceof RateLimitingError ? err.rateLimitReset : Number(err?.headers?.["x-ratelimit-reset"]);
-		const delay = Math.max(rateLimit * 1000 - Date.now(), 0);
+	// TODO: rethrow as TaskError and handle in SQS error handler
+	if (err instanceof RateLimitingError) {
```

Comment:
What happen to `|| Number(err?.headers?.["x-ratelimit-remaining"]) == 0;`

Reply:
This is some old crap when we had octokit. We don't use it for a long time already, in github client all exceptions are wrapped into GitHubClientError and others